### ISSUE TITLE
Text/Multimedia seach engine

### ISSUE THREAD

**Issue by cosmologist10** [ILUGD/talks#68]
**Abstract**
	A search engine for user text and multimedia contents. This talk is divided into two parts:
1. For text files: It will lookup for a particular word in the given directory.
2. For multimedia files: It will lookup in the given directory, according to the specified metadata for images/audio/video files.
Multiprocessing would be used to increase the efficiency of searching.
       
**Expected Duration**
        25 minutes to 30 minutes

**Track**
    	Beginner

**Pre-requisite**
   	Python, argparser, metadata, exiftool, cPickle, unittest
        
**Resources**
    https://github.com/cosmologist10/text-multimedia-search-engine-for-user-contents

**Slides**
    Will be updated soon.

**Comment by sourabhtk37** [comment]
Lovely. Looking into [fzf](https://github.com/junegunn/fzf) too.

**Comment by rajudev** [comment]
@sourabhtk37 are we confirming this for upcoming meetup?

**Comment by sourabhtk37** [comment]
We need to polish the talk a bit to be interesting for the audience.
@cosmologist10 Can you illustrate how the it works at shell level since the argparse and subprocess module speaks with the shell and the subprocess. 
Also please let us know your availabilty on 21st of January.

**Comment by cosmologist10** [comment]
Yeah, I am working on the presentation and slides part with few changes in the code base, It will explain how things are interacting on shell level, and uses of subprocess module, etc. It will get updated by Thursday.

Yeah, I will be available on 21st of January.
